libff/algebra/curves/bn128/bn128_g1.hpp

```diff
@@ -33,11 +33,15 @@ class bn128_G1 {
     static bn128_G1 G1_zero;
     static bn128_G1 G1_one;
 
-    bn::Fp coord[3];
-    bn128_G1();
     typedef bn128_Fq base_field;
     typedef bn128_Fr scalar_field;
 
+    bn::Fp X, Y, Z;
+    void fill_coord(bn::Fp (*coord)[3]) const { (*coord)[0] = this->X; (*coord)[1] = this->Y; (*coord)[2] = this->Z; return; };
```

Comment:
I think these methods can take a `bn::Fp *` (as `bn::Fp coord[3]`) rather than `bn::Fp **`.
I tried the change in https://github.com/clearmatics/libff/pull/6.

Reply:
Thanks @dtebbs just merged your PR